.github/dependabot.yml

```diff
@@ -66,3 +66,7 @@ updates:
   - dependency-name: y18n
     versions:
     - 4.0.1
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
```

Comment:
Hi @boahc077,  this is an awesome addition :+1: , was there a issue for this? if so please link it on the description. Also was thinking maybe we can start with a monthly interval see how that goes and maybe move it to weekly if that goes well? What do you think? Thanks